modules/ROOT/pages/bundling-plugins.adoc

```diff
@@ -7,17 +7,11 @@
 include::partial$module-loading/bundling-ref-example.adoc[]
 :!editorcomponent:
 
-The following table shows examples of the syntax used to bundle the following plugin.
-
-Required files for an example plugin:
+include::partial$plugin-files/plugin-file-index.js.adoc[]
 
-[source, js]
-----
-./plugins/example/content.js // or './plugins/example/content_css.js'
-./plugins/example/plugin.js
-----
+The following table shows examples of the syntax used to bundle the following plugin.
```

Comment:
```suggestion
The following table shows examples of the syntax used to bundle the "<plugincode>" plugin.
```

Reply:
Now that I think about it, do we need this paragraph if we have one below noting the table?

Reply:
Good point, lets remove this paragraph

Reply:
```suggestion
```